docs/src/main/sphinx/release/release-454.md

```diff
@@ -0,0 +1,88 @@
+# Release 454 (7 Aug 2024)
+
+## General
+
+* Add support for retrieving lightweight query information from the `QueryInfo`
+  endpoint with the new `pruned` property. ({issue}`20843`)
+* Improve performance of queries with multiple distinct aggregations. ({issue}`22355`)
+* Skip query completed notifications in the event listener when there are too
+  many concurrent completion requests. The maximum number of requests to handle
+  before skipping can be configured with the
+  `event-listener.max-concurrent-query-completed-events` configuration property. ({issue}`22911`)
+* Fix failure for queries involving {func}`json_parse` and a cast to `array`,
+  `map`, or `row`. ({issue}`22840`)
+
+## Security
+
+* Fix the Trino user incorrectly defaulting to the `user.name` property when no
+  user is provided. ({issue}`22903`)
+
+## Web UI
+
+* Add information about which tables and routines have been referenced by a
+  query. ({issue}`20843`)
+
+## JDBC driver
+
+* Add support for the `NO_GENERATED_KEYS` option in the
+  `Connection.prepareStatement` method. ({issue}`22100`)
+
+## BigQuery connector
+
+* Add support for the `execute` procedure. ({issue}`22692`)
+
+## Delta Lake connector
+
+* Add S3 security mapping for the native S3 filesystem. ({issue}`22559`)
+* Add support for caching table metadata to the metastore. ({issue}`21463`)
+* {{breaking}} Rename metastore OpenTelemetry attributes from `trino.hive.*` to
+  `trino.metastore.*`. ({issue}`22866`)
+
+## Hive connector
+
+* Add support for using the `hive.metastore-refresh-interval` configuration
+  property when using Glue v2 Hive metastore. ({issue}`22706`)
+* Add S3 security mapping for the native S3 filesystem. ({issue}`22559`)
+* {{breaking}} Rename metastore OpenTelemetry attributes from `trino.hive.*` to
+  `trino.metastore.*`. ({issue}`22866`)
+* Add support for adjusting UTC-adjusted `INT64` timestamps to the timezone
+  specified in the `hive.parquet.time-zone` property when reading Parquet files. ({issue}`22577`)
+* Fix potential query failure caused by incorrect bucket column validation. ({issue}`22261`)
+* Fix error when reading `INT96` timestamps in Parquet files generated by AWS
+  Wrangler. ({issue}`22854`)
+
+## Hudi connector
+
+* Add S3 security mapping for the native S3 filesystem. ({issue}`22559`)
+* {{breaking}} Rename metastore OpenTelemetry attributes from `trino.hive.*` to
+  `trino.metastore.*`. ({issue}`22866`)
+
+## Iceberg connector
+
+* Add support for the [Polaris catalog](https://polaris.io/). ({issue}`22909`)
```

Comment:
That PR doesn't add support for Polaris. It just adds some tests for existing behavior. Remove this item.